pkg/server/build.go

```diff
@@ -8,5 +8,9 @@ import (
 
 func (ctrl *Controller) buildHandler(w http.ResponseWriter, _ *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
-	_, _ = w.Write([]byte(build.PrettyJSON()))
+	_, err := w.Write([]byte(build.PrettyJSON()))
+
+	if err != nil {
+		ctrl.log.WithError(err).Error("Build Handler failed")
+	}
```

Comment:
By convention, log messages should not begin with a capital letter (except acronyms). Logging an error without a proper annotation might be confusing – consider "writing response body" or "could not write response".

By the way, there are very limited number of cases when `http.ResponseWriter` (`*http.response`, actually) returns an error. In vast majority of them those are related to network (e.g., client closed connection) and usually are suppressed.